### PR TITLE
Improve continuous agg user messages

### PR DESCRIPTION
**Pull request by cevian**
Switch from using internal timestamps to more user-friendly
timestamps in our log messages and clean up some messages.

### CODE REVIEW

File: tsl/src/continuous_aggs/materialize.c [hunk 1]
```diff
@@ -421,11 +421,11 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 		if (verbose)
 			elog(INFO,
 				 "new materialization range not found for %s.%s (time column %s): "
-				 "not enough new data past completion threshold (" INT64_FORMAT ")",
+				 "not enough new data past completion threshold as of %s",
```
Comment (gayyappan) [inline tsl/src/continuous_aggs/materialize.c]: this looks a little strange with integer messages like : not enough new data past completion threshold as of 12. I think the older formatting "not enough new data past completion threshold(12)" looks better.

File: tsl/src/continuous_aggs/materialize.c [hunk 2]
```diff
@@ -396,11 +396,11 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 		if (verbose)
 			elog(INFO,
 				 "new materialization range not found for %s.%s (time column %s): not enough data "
-				 "in table (" INT64_FORMAT ")",
+				 "past minimum time value as of %s",
```
Comment (gayyappan) [inline tsl/src/continuous_aggs/materialize.c]: this should be reworded. suggest we use some terms from the docs for the messages. suggestion "past materialization threshold as of %s".

File: tsl/src/continuous_aggs/materialize.c (hunk 1 above)
Comment (cevian) [reply tsl/src/continuous_aggs/materialize.c]: Fixed

File: tsl/src/continuous_aggs/materialize.c (hunk 2 above)
Comment (cevian) [reply tsl/src/continuous_aggs/materialize.c]: This is actually the correct message since it deals with underflow not any of the thresholds

Comment (gayyappan) [reply tsl/src/continuous_aggs/materialize.c]: How about rewording it as: " new materialization range not found ....: not enough data that **satisfies refresh lag criterion** as of ..."? ( ***minimum time value*** does not make much sense to end user?)
